config/files/usr/etc/NetworkManager/conf.d/rand_mac.conf

```diff
@@ -3,6 +3,6 @@
 wifi.scan-rand-mac-address=yes
  
 [connection-mac-randomization]
-# Generate a random MAC for each Network and associate the two permanently.
-ethernet.cloned-mac-address=stable
-wifi.cloned-mac-address=stable
+# generate a random mac address per each connection of the network
+ethernet.cloned-mac-address=random
+wifi.cloned-mac-address=random
```

Comment:
very much. Any mac-based filter will get completely spammed which can break home networks and cause a lot of trouble. We need GUI toggles in DEs for this, then this can be added.